archive/extract_historical_data.py

```diff
@@ -1,18 +1,69 @@
-# This script is a work in progress. Please feel free to use it as a starting point.
-import json
+#!/usr/bin/env python
 
+import os
+import json
+import re
 import pandas as pd
+import importlib
+job_title = importlib.import_module("job_title", package=None)
+
+#dir path of input json files
+input_dir = './archive'
+json_files = []
+final_df = pd.DataFrame()
+
+#list all json in input dir
+for file in os.listdir(input_dir):
+    if file.endswith(".json"):
+        json_files.append(os.path.join(input_dir, file))
```

Comment:
```suggestion
import glob

json_files = glob.glob(os.path.join(input_dir, '*.json'))
```
This way is completely fine and understandable I just wanted to show an alternate way of getting all of the files in a directory.